network-api/networkapi/donate/tests/customblocks/test_notice_block.py

```diff
@@ -0,0 +1,55 @@
+from django.test import TestCase
+from wagtail import rich_text
+from wagtail.blocks.struct_block import StructBlockValidationError
+from wagtail.images.tests.utils import Image, get_test_image_file
+
+from networkapi.donate.pagemodels.customblocks.notice_block import NoticeBlock
+
+
+class NoticeBlockTest(TestCase):
+    def setUp(self):
+        self.notice_block = NoticeBlock()
+
+    def test_valid_notice_block(self):
+        value = {
+            "image": Image.objects.create(title="Test Image", file=get_test_image_file()),
+            "image_alt_text": "Alt text",
+            "text": rich_text.RichText("<p>Some content</p>"),
+        }
+        # If clean method does not raise ValidationError, the test will pass
```

Comment:
I think it is better to be explicit here, rather than relying on something not happening...

Reply:
Thanks for the feedback Jhonatan! I have updated these tests with `assert` statements, and added a new one for the new `NoticeBlockFactory`